### PR TITLE
SimpleITK 0.9.1 update

### PR DESCRIPTION
**Pull request by blowekamp**
Adding more specific library dependencies to avoid missing libraries at runtime. There appears to be no check for matching library so version between build time and run time if not manually specified.

### CODE REVIEW

File: simpleitk/bld.bat [hunk 1]
```diff
@@ -1,8 +1,12 @@
 mkdir C:\b
 cd C:\b
 
-if "%ARCH%" == "32" set "CMAKE_GENERATOR=Visual Studio 9 2008"
-if "%ARCH%" == "64" set "CMAKE_GENERATOR=Visual Studio 9 2008 Win64"
+set "CMAKE_GENERATOR=Visual Studio 9 2008"
+if "%PY_VER%" == "3.3" set "CMAKE_GENERATOR=Visual Studio 10"
+if "%PY_VER%" == "3.4" set "CMAKE_GENERATOR=Visual Studio 10"
+if "%PY_VER%" == "3.5" set "CMAKE_GENERATOR=Visual Studio 13"
```
Comment (ccordoba12) [inline simpleitk/bld.bat]: These are

* `Visual Studio 10 2010`
* `Visual Studio 14 2015`

according to this page

https://cmake.org/cmake/help/v3.3/manual/cmake-generators.7.html

Comment (blowekamp) [inline simpleitk/bld.bat]: Thanks. You are correct  with the Visual Studio 14.

The years are optional for recent versions of Visual Studio.

**Comment by ccordoba12** [comment]
Thanks for the fix :-)

**Comment by blowekamp** [comment]
Thank you for the prompt merge!!

**Pull request merged**